ccp/src/prover/tests.rs

```diff
@@ -143,6 +143,8 @@ async fn prover_on_active_no_active_commitment() {
 
 #[test(tokio::test(flavor = "multi_thread", worker_threads = 3))]
 async fn prover_on_active_reduce_allocation_on_active_commitment() {
+    let _ = env_logger::builder().is_test(true).try_init();
```

Comment:
It shouldn't not be needed as the test is decorated with `test_log::test` macro.

Reply:
Good point